src/tests/simulation/src/risingwave.toml

```diff
@@ -2,6 +2,10 @@
 #
 # Note: this file is embedded in the binary and cannot be changed without recompiling.
 
+[meta]
+# a relatively small number to make it easier to timeout
+max_heartbeat_interval_secs = 10
```

Comment:
Should we maybe go smaller? We use have a timeout of 10s for a meta leader lease in the [example.toml](https://github.com/risingwavelabs/risingwave/blob/43c81b9275d70167ccc16055a09fdece5f6e3d1b/src/config/example.toml#L17). If we use that here we may not get a timeout.